crates/block-producer/src/mempool/batch_graph.rs

```diff
@@ -278,6 +282,17 @@ mod tests {
         assert_eq!(err, expected);
     }
 
+    #[test]
+    fn insert_with_internal_parent_succeeds() {
+        //! Ensure that a batch with internal dependencies can be inserted.
```

Comment:
nit: this should be a simple comment (i.e., `//`), right?

Reply:
I was hoping we could somehow generate docs for these, allowing us to browse tests and their intentions via `cargo doc`. Seemed like a nice way to get an overview of what's been tested and how.

Unfortunately doesn't work this way; no way to generate these that I could find.